enterprise/views.py

```diff
@@ -1025,7 +1025,10 @@ def extend_course(course):
         * `staff`
         """
         try:
-            catalog_api_client = CourseCatalogApiServiceClient()
+            if enterprise_customer is not None:
+                catalog_api_client = CourseCatalogApiServiceClient(enterprise_customer.site)
+            else:
+                catalog_api_client = CourseCatalogApiServiceClient()
```

Comment:
Let it raise an exception if `enterprise_customer` is None.

Reply:
ack